### ISSUE TITLE
Stale shares increasing

### ISSUE THREAD

**Issue by barret71691** [nicehash/NiceHashQuickMiner#319]
Hi Hashers,

I am having trouble with one of my rigs that is showing an increasing amount of stale shares in the past day. Below shows 2.69 percent in the last day.
![A](https://user-images.githubusercontent.com/80551450/113443945-e894c580-93c0-11eb-820c-2a18a8a2f462.png)

Before yesterday the percentage was pretty low.
![B](https://user-images.githubusercontent.com/80551450/113443995-03ffd080-93c1-11eb-9bd3-1481a2106367.JPG)

I do have other rigs on the same network and are not showing these spikes, below is an example.
![C](https://user-images.githubusercontent.com/80551450/113444090-33aed880-93c1-11eb-92f9-452ad0d839fe.JPG)

I have tried changing the location on the config file to all that are available, as well as -1 which was the default. Other basic things I did were restarting computer, restarting my connections, etc.

Doesn't seem to be related to the latest update since the other rigs are working fine, they are all 3080s. Only other thing I can think of is an issue with the motherboard wi-fi card? Any thoughts/idea would be helpful.

Thanks,

**Comment by barret71691** [comment]
Issue resolved! The particular rig was automatically connecting to the wrong wireless network. Rejected shares are now virtually non-existent.

**Comment by barret71691** [comment]
Update: the issue came back

The resolution was specific to the MS I was getting per accepted share. So it would be like 200-300 MS per share and once I changed the wireless to the main connection, it dropped to the expected 20 - 30 MS per accepted share.

I was still getting rejected share and this wasn't always the case. The only new thing I added was another rig, so I ended up hard-wiring that rig instead of using wireless and it seems (so far) to have resolved the issue. I will come back to update here with results. Seems like there was interference of some sorts or too many connections.
